Movie/Movie/Presentation/DetailsMovie/Presenter/DetailsMoviePresenterProtocol.swift

```diff
@@ -6,11 +6,12 @@ import Foundation
 /// Вход экрана выбора фильмов
 protocol DetailsMoviePresenterProtocol: AnyObject {
     var movieDetails: MovieDetails? { get set }
-    var movieId: Int? { get set }
+    var id: Int? { get set }
     var recommendationMovies: [RecommendationMovie] { get set }
-    var networkService: NetworkServiceProtocol { get }
+    var dataProvider: DataProviderProtocol { get }
```

Comment:
```suggestion
    var dataProvider: DataProviderProtocol { get }
```
Лучше называть все сервисами а не Provider